ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.13.yaml

```diff
@@ -69,8 +69,15 @@ tests:
   steps:
     cluster_profile: azure-qe
     env:
-      ARO_CLUSTER_VERSION: 4.11.26
+      ARO_CLUSTER_VERSION: 4.13.40
     workflow: cucushift-installer-rehearse-azure-aro
+- as: installer-rehears-azure-aro-private
+  cron: '@yearly'
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ARO_CLUSTER_VERSION: 4.13.40
+    workflow: cucushift-installer-rehearse-azure-aro-private
```

Comment:
Just one minor comment, suggest to replace the workflow at line 73 with this one to save one more job.